aten/src/ATen/native/cuda/UnarySignKernels.cu

```diff
@@ -52,21 +53,45 @@ void signbit_kernel_cuda(TensorIteratorBase& iter){
 }
 
 template<typename T>
-__host__ __device__ static inline c10::complex<T> sgn_wrapper(c10::complex<T> z) {
+C10_HOST_DEVICE static inline c10::complex<T> sgn_wrapper(c10::complex<T> z) {
   if (z == c10::complex<T>(0, 0)) {
     return c10::complex<T>(0, 0);
   } else {
     return z / std::abs(z);
   }
 }
 
+const char sgn_name[] = "sgn_kernel";
 void sgn_kernel_cuda(TensorIteratorBase& iter){
-  AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "sgn_cuda", [&]() {
+  auto dtype = iter.dtype();
+  #if AT_USE_JITERATOR()
+    static const auto sgn_string = jiterator_stringify(
+        template <typename T>
+        T sgn_kernel(T a) {
+          const T z = T(0);
+	  if (a == z) {
+	    return z;
+	  } else {
+	    return z / std::abs(a);
```

Comment:
Should be a / abs(a).

Reply:
Hi, @kshitij12345!
Is there any reason for not using `std::` here? The wrapper contains that, so I thought to use the same.

> Should be a / abs(a).

Here `z` -> `a`: fixed this typo (made mistake while changing the variable name, locally) Thanks for pointing that out! :)

Thanks! :)

Reply:
> Is there any reason for not using std:: here?

Where are you referring to?